src/ui/Templates/CoreHelpers.ts

```diff
@@ -297,15 +312,36 @@ export interface ICoreHelpers {
 
 /**
  * Available options for the size templateHelpers.
+ *
+ * Example:
+ * <div class="CoveoFieldValue" data-helper="helperName" data-helper-options-optionName="option-value"></div>
  */
 export interface ISizeOptions {
   /**
    * The base into which to format the value.
+   * Formula: value * 10^(3 * base)
+   *
+   * Base 0 Examples:
```

Comment:
For consistency's sake, maybe you could format these examples as such:

**Examples:**
> **Base 0:**
> `1 => 1B`
> `1000 => 1MB`
>
> **Base 1:**
> `1 => 1MB`
> `1000 => 1GB`

I'd do the same for the `precision` option examples.